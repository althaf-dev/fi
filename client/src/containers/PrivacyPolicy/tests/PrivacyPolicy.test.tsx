/* eslint-disable no-undef, import/no-extraneous-dependencies */
import React from 'react';
import { expect } from '@jest/globals';

import { render } from '../../../testing/test-utils';
import PrivacyPolicyHome from '../index';

it('should render Privacy Policy component', () => {
    const { asFragment } = render(<PrivacyPolicyHome />);
    expect(asFragment()).toMatchSnapshot();
});
