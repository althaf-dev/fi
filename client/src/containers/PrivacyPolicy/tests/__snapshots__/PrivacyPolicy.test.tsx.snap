// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Privacy Policy component 1`] = `
<DocumentFragment>
  <div
    class="sc-ckMVTt dYOwIC"
  >
    <div
      class="sc-fbPSWO bwNDJV"
    >
      <div
        class="sc-ehmTmK jFzFdk"
      >
        <div
          class="sc-olbas sc-hiMGwR IdaQF bIFTrT"
        >
          <div
            class="sc-lbOyJj lemKKq"
          >
            <img
              alt="Fi"
              class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
              loading="eager"
              src="logo.svg"
            />
          </div>
          <div
            class="sc-eKszNL dvXZdg"
          >
            <div
              class="sc-eCYdqJ cnGEXz sc-gSAPjG hJBHXF"
              color="GAINSBORO"
            >
              <div
                class="sc-iNWwEs eKIqhq"
              >
                About Us
                <div
                  class="sc-BeQoi BPuHZ"
                >
                  <div
                    class="sc-kgUAyh gXSeJt"
                  >
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                  </div>
                </div>
              </div>
              <div
                class="sc-iNWwEs eKIqhq"
              >
                Product
                <div
                  class="sc-BeQoi BPuHZ"
                >
                  <div
                    class="sc-kgUAyh gXSeJt"
                  >
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                  </div>
                </div>
              </div>
              <div
                class="sc-iNWwEs eKIqhq"
              >
                <div />
              </div>
              <div
                class="sc-iNWwEs eKIqhq"
              >
                Resources
                <div
                  class="sc-BeQoi BPuHZ"
                >
                  <div
                    class="sc-kgUAyh gXSeJt"
                  >
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <span
                        class="sc-jgbSNz lamefd"
                      >
                        <a
                          href="/blog"
                        >
                          Blog
                        </a>
                      </span>
                    </h3>
                    <h3
                      class="sc-kDDrLX kUqEAz"
                      color="CHARCOAL_GREY"
                    >
                      <div />
                    </h3>
                  </div>
                </div>
              </div>
              <div
                class="sc-iNWwEs eKIqhq"
              >
                <div />
              </div>
            </div>
            <div
              class="sc-iAvgwm fvwLWt"
            >
              <div
                class="sc-efBctP fqSSbl"
              >
                <div>
                  <div
                    class="sc-iTONeN sc-cTQhss hPlKkv jSgNfZ"
                  >
                    <img
                      alt="logo"
                      class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                      loading="lazy"
                      src="prodQRCode.png"
                    />
                  </div>
                  <div
                    class="sc-eCYdqJ fxZcwK sc-bBXxYQ hMmiBy"
                    color="MINE_SHAFT"
                  >
                    Scan QR to get the Fi app
                  </div>
                  <div
                    class="sc-cOFTSb ecfeHO"
                  >
                    <a
                      href="https://play.google.com/store/apps/details?id=com.epifi.paisa&referrer=utm_source%3Dfimoney"
                      id="qrcodescanner_playstore_link"
                      rel="noreferrer"
                      target="_blank"
                    >
                      <div
                        class="sc-iTONeN sc-jOrMOR hPlKkv eVGPt"
                      >
                        <img
                          alt="logo"
                          class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                          loading="lazy"
                          src="play-store_black.png"
                        />
                      </div>
                    </a>
                    <a
                      href="https://apps.apple.com/in/app/fi-money-digital-banking/id1531564767"
                      id="qrcodescanner_appstore_link"
                      rel="noreferrer"
                      target="_blank"
                    >
                      <div
                        class="sc-iTONeN sc-dPyBCJ hPlKkv fvEJLU"
                      >
                        <img
                          alt="logo"
                          class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                          loading="lazy"
                          src="app-store_black.png"
                        />
                      </div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-TRNrF jjodjB"
      >
        <div
          class="sc-jIAOiI drAXQh"
        >
          <img
            alt="Fi"
            class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
            loading="eager"
            src="logo.svg"
          />
        </div>
        <div
          class="sc-eKszNL dvXZdg"
        >
          <div
            class="sc-eCYdqJ TqLYS sc-gSAPjG hJBHXF"
            color="SILVER_2"
          >
            <div
              class="sc-iNWwEs eKIqhq"
            >
              About Us
              <div
                class="sc-BeQoi BPuHZ"
              >
                <div
                  class="sc-kgUAyh gXSeJt"
                >
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                </div>
              </div>
            </div>
            <div
              class="sc-iNWwEs eKIqhq"
            >
              Product
              <div
                class="sc-BeQoi BPuHZ"
              >
                <div
                  class="sc-kgUAyh gXSeJt"
                >
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                </div>
              </div>
            </div>
            <div
              class="sc-iNWwEs eKIqhq"
            >
              <div />
            </div>
            <div
              class="sc-iNWwEs eKIqhq"
            >
              Resources
              <div
                class="sc-BeQoi BPuHZ"
              >
                <div
                  class="sc-kgUAyh gXSeJt"
                >
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <span
                      class="sc-jgbSNz lamefd"
                    >
                      <a
                        href="/blog"
                      >
                        Blog
                      </a>
                    </span>
                  </h3>
                  <h3
                    class="sc-kDDrLX kUqEAz"
                    color="CHARCOAL_GREY"
                  >
                    <div />
                  </h3>
                </div>
              </div>
            </div>
            <div
              class="sc-iNWwEs eKIqhq"
            >
              <div />
            </div>
          </div>
          <div
            class="sc-iAvgwm fvwLWt"
          >
            <div
              class="sc-efBctP fqSSbl"
            >
              <div>
                <div
                  class="sc-iTONeN sc-cTQhss hPlKkv jSgNfZ"
                >
                  <img
                    alt="logo"
                    class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                    loading="lazy"
                    src="prodQRCode.png"
                  />
                </div>
                <div
                  class="sc-eCYdqJ fxZcwK sc-bBXxYQ hMmiBy"
                  color="MINE_SHAFT"
                >
                  Scan QR to get the Fi app
                </div>
                <div
                  class="sc-cOFTSb ecfeHO"
                >
                  <a
                    href="https://play.google.com/store/apps/details?id=com.epifi.paisa&referrer=utm_source%3Dfimoney"
                    id="qrcodescanner_playstore_link"
                    rel="noreferrer"
                    target="_blank"
                  >
                    <div
                      class="sc-iTONeN sc-jOrMOR hPlKkv eVGPt"
                    >
                      <img
                        alt="logo"
                        class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                        loading="lazy"
                        src="play-store_black.png"
                      />
                    </div>
                  </a>
                  <a
                    href="https://apps.apple.com/in/app/fi-money-digital-banking/id1531564767"
                    id="qrcodescanner_appstore_link"
                    rel="noreferrer"
                    target="_blank"
                  >
                    <div
                      class="sc-iTONeN sc-dPyBCJ hPlKkv fvEJLU"
                    >
                      <img
                        alt="logo"
                        class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                        loading="lazy"
                        src="app-store_black.png"
                      />
                    </div>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-ktCSKO JsBAF sc-gYMRRK gIJoSD"
      >
        <div
          class="sc-bGWzfD cxgLXy"
        >
          <div
            class="sc-IIEeM jCIzsX"
          >
            <h1
              class="sc-papXJ fEfRcZ sc-gGnURB gzxXgY"
              color="WHITE"
            >
              Privacy Policy
            </h1>
            <div
              class="sc-eCYdqJ jVNGkM sc-iuStju jPsvXh"
              color="SILVER_2"
            >
              Epifi Technologies Private Limited (epiFi, we, us or Company) is a fintech company providing financial solutions and services. We believe that our customers (You) deserve a ‘no shenanigans’ approach to their Personal Data.
              <br />
              <br />
              By and large, this policy has minimized the usage of legal jargon – unless absolutely required on a lawful basis – and explains how we treat your information.
            </div>
          </div>
          <div
            class="sc-fytwQQ cTtcGf"
          >
            <img
              alt="image"
              class="sc-gsnTZi sc-dkzDqf krIvpr eoDeNX"
              loading="lazy"
              src="privacy-policy-mobile-poster.webp"
              width="auto"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-fejtnb fpGptu"
    >
      <div>
        <div
          class="sc-coCPJf sc-bcnBk bhNlI llCdxP"
        >
          <div
            class="sc-hCDzWh bXfKDn"
          >
            <span>
              <div
                class="sc-eCYdqJ iidbqz sc-jUtGcO sc-keNpes cTbcVF kZEUXy"
                color="SUVA_GREY"
              >
                We aim for the highest standards of safety, security and confidentiality
    when using your data, and this policy describes how we collect and use your information
    and discusses this in detail in the below mentioned sections:
              </div>
            </span>
            <div
              class="sc-cmYsgE bpRitP"
            >
              <div
                class="sc-eCYdqJ fxZcwK sc-cBOWjd gHgjAZ"
                color="MINE_SHAFT"
              >
                Definitions
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                Applicability of this policy
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                What data do we collect and why?
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                How is your information used?
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                Who do we share your data with?
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                How do we secure your data?
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                Know your rights
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
              <div
                class="sc-eCYdqJ cCftFk sc-cBOWjd gHgjAZ"
                color="FOREST_GREEN"
              >
                Policy upgrades and changes
                <div
                  class="sc-eQNgno eOwtjE"
                />
              </div>
            </div>
            <div
              class="sc-hGtivm eTqiTs"
            />
          </div>
        </div>
        <div
          class="sc-iNFqmR eqEGLH"
        >
          <div
            class="sc-ePsPkC hTowoT"
          >
            <div
              class="sc-eCYdqJ iidbqz sc-jUtGcO sc-eCDnqH cTbcVF iIYthI"
              color="SUVA_GREY"
            >
              Long story short, anytime you register on our website/app, you are required to comply with our Privacy Policy.
              <br />
              We will use your data for legitimate purposes only while safeguarding your privacy concerns.
            </div>
          </div>
        </div>
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
      </div>
    </div>
    <div
      class="sc-cwpsFg KVyBC"
    >
      <div
        class="sc-dkSuNL eCmbdT"
      >
        <div
          class="sc-bczRLJ sc-kngDgl sc-gJwTLC cFvSHK TlBtU jWiOvF"
        >
          <div
            class="sc-fxvKuh iLEuH"
          >
            <div
              class="sc-iJkHyd dEepvI"
            >
              <div>
                <div
                  class="sc-eCYdqJ bnCuWB"
                  color="FI_GREEN"
                >
                  Your Privacy.
                  <br />
                  Minus the jargon.
                </div>
              </div>
              <div
                class="sc-iWajrY czNAel"
              >
                <span
                  class="sc-ftvSup gsYrLz"
                  color="WHITE"
                >
                  <img
                    alt="arrow"
                    class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                    loading="eager"
                    src="top-right-arrow.svg"
                  />
                </span>
              </div>
            </div>
            <div
              class="sc-iJkHyd ejKlTU"
            >
              <div>
                <div
                  class="sc-eCYdqJ cMbRGg"
                  color="FI_PINK"
                >
                  No Hidden
                  <br />
                  Fees!
                </div>
              </div>
              <div
                class="sc-iWajrY czNAel"
              >
                <span
                  class="sc-ftvSup gsYrLz"
                  color="WHITE"
                >
                  <img
                    alt="arrow"
                    class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                    loading="eager"
                    src="top-right-arrow.svg"
                  />
                </span>
              </div>
            </div>
            <a
              href="https://jobs.lever.co/epifi"
              rel="noreferrer"
              target="_blank"
            >
              <div
                class="sc-iJkHyd ejKlTU"
              >
                <div>
                  <div
                    class="sc-eCYdqJ jdNfrJ"
                    color="FI_LIGHT_BLUE"
                  >
                    Join the
                    <br />
                    team.
                  </div>
                </div>
                <div
                  class="sc-iWajrY czNAel"
                >
                  <span
                    class="sc-ftvSup gsYrLz"
                    color="WHITE"
                  >
                    <img
                      alt="arrow"
                      class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                      loading="eager"
                      src="top-right-arrow.svg"
                    />
                  </span>
                </div>
              </div>
            </a>
          </div>
          <div
            class="sc-jKDlA-D bKqmIr"
          >
            <img
              alt="image"
              class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
              loading="lazy"
              src="footer-logo.svg"
            />
          </div>
          <div
            class="sc-bczRLJ sc-bTMaZy cFvSHK yQEXi"
          >
            <div
              class="sc-bczRLJ sc-afnQL sc-tQuYZ cFvSHK lewnTt ferVki"
            >
              <div
                class="sc-dSuTWQ kPVxin"
              >
                <div
                  class="sc-eCYdqJ fzRvzi"
                  color="TEXT_GREY_1"
                >
                  Pronounced 
                  <span
                    class="sc-ftvSup gOxMIH"
                    color="WHITE"
                  >
                     Fī(-ē) 
                  </span>
                  and sounds like
                  <div
                    class="sc-kjEcyX ikirKI"
                  >
                    <img
                      alt="volume"
                      class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                      loading="lazy"
                      src="volume-active.svg"
                    />
                  </div>
                  <audio
                    class="sc-cLFqLo jDIWhf"
                  >
                    <source
                      src="FiAudio.mp3"
                      type="audio/mpeg"
                    />
                  </audio>
                </div>
              </div>
              <div
                class="sc-bczRLJ sc-kngDgl sc-cHPgQl cFvSHK TlBtU bYzimd"
              >
                <div
                  class="sc-fHsOPI jpBIgo"
                >
                  <div
                    class="sc-gkJlnC iforIl"
                  >
                    hi
                  </div>
                  <div
                    class="sc-gzzPqb fnEVGI"
                  >
                    <img
                      alt="image"
                      class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                      loading="lazy"
                      src="hifi.png"
                    />
                  </div>
                </div>
                <div
                  class="sc-fHsOPI jpBIgo"
                >
                  <div
                    class="sc-gkJlnC iforIl"
                  >
                    sky
                  </div>
                  <div
                    class="sc-gzzPqb fnEVGI"
                  >
                    <img
                      alt="image"
                      class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                      loading="lazy"
                      src="sky.png"
                    />
                  </div>
                </div>
                <div
                  class="sc-fHsOPI jpBIgo"
                >
                  <div
                    class="sc-gkJlnC iforIl"
                  >
                    tie
                  </div>
                  <div
                    class="sc-gzzPqb fnEVGI"
                  >
                    <img
                      alt="image"
                      class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                      loading="lazy"
                      src="tie.svg"
                    />
                  </div>
                </div>
                <div
                  class="sc-fHsOPI jpBIgo"
                >
                  <div
                    class="sc-gkJlnC iforIl"
                  >
                    fly
                  </div>
                  <div
                    class="sc-gzzPqb fnEVGI"
                  >
                    <img
                      alt="image"
                      class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
                      loading="lazy"
                      src="fly.svg"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div>
              <div
                class="sc-bczRLJ sc-gHLcSH cFvSHK kEhzMq"
              >
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
              </div>
              <div
                class="sc-bczRLJ sc-djUGQo cFvSHK cHDGYA"
              >
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <a
                    href="/blog"
                  >
                    Blog
                  </a>
                </h3>
                <h3
                  class="sc-kDDrLX eMDbJf sc-jhzXDd coOizB"
                  color="WHITE"
                >
                  <div />
                </h3>
              </div>
            </div>
          </div>
          <div
            class="sc-bczRLJ sc-hZFzCs cFvSHK gaLYiB"
          >
            <div
              class="sc-bczRLJ sc-afnQL sc-hjQCSK cFvSHK lewnTt kMduYt"
            >
              <div
                class="sc-eCYdqJ hdRMJn sc-bSakgD jyOxpN"
                color="TEXT_GREY_1"
              >
                Contact Fi Money customer care
              </div>
              <div
                class="sc-bczRLJ sc-kngDgl sc-jvLaUc cFvSHK TlBtU kjFwxX"
              >
                <div
                  class="sc-fmrZth fmrhOc"
                >
                  <div
                    class="sc-eXBvqI gyNIoe"
                  >
                    080-47485490
                  </div>
                </div>
                <div
                  class="sc-fmrZth fmrhOc"
                >
                  <a
                    href="mailto:help@fi.care"
                  >
                    <div
                      class="sc-eXBvqI gyNIoe"
                    >
                      help@fi.care
                    </div>
                  </a>
                </div>
                <div
                  class="sc-fmrZth fmrhOc"
                >
                  <div
                    class="sc-eXBvqI gyNIoe"
                  >
                    In-app chat
                  </div>
                </div>
              </div>
              <div
                class="sc-bczRLJ cFvSHK"
              >
                <div
                  class="sc-edUIhV hrYfin"
                >
                  <a
                    href="https://www.instagram.com/bankonfi"
                    rel="noreferrer"
                    target="_blank"
                  >
                    <img
                      alt="instagram"
                      class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                      loading="lazy"
                      src="instagram-white.svg"
                    />
                  </a>
                </div>
                <div
                  class="sc-edUIhV hrYfin"
                >
                  <a
                    href="https://www.twitter.com/Bank_on_Fi"
                    rel="noreferrer"
                    target="_blank"
                  >
                    <img
                      alt="twitter"
                      class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                      loading="lazy"
                      src="twitter-white.svg"
                    />
                  </a>
                </div>
                <div
                  class="sc-edUIhV hrYfin"
                >
                  <a
                    href="https://www.linkedin.com/company/epifi"
                    rel="noreferrer"
                    target="_blank"
                  >
                    <img
                      alt="linkedin"
                      class="sc-gsnTZi sc-hKMtZM krIvpr hFZcsr"
                      loading="lazy"
                      src="linkedin-white.svg"
                    />
                  </a>
                </div>
              </div>
              <div
                class="sc-bczRLJ sc-jmnVvD cFvSHK dGHkGt"
              >
                <div
                  class="sc-eCYdqJ hdRMJn sc-iXxrte gxfcTv"
                  color="TEXT_GREY_1"
                >
                  <div />
                </div>
                <div
                  class="sc-eCYdqJ hdRMJn sc-iXxrte gxfcTv"
                  color="TEXT_GREY_1"
                >
                  <div />
                </div>
                <div
                  class="sc-eCYdqJ hdRMJn sc-iXxrte gxfcTv"
                  color="TEXT_GREY_1"
                >
                  <a
                    href="https://www.federalbank.co.in/our-commitments"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Partner Bank Codes & Policies
                  </a>
                </div>
              </div>
            </div>
            <div
              class="sc-hZgfyJ eWgCsw"
            >
              <div
                class="sc-bczRLJ sc-hKpBwk cFvSHK loyOHF"
              >
                <p
                  class="sc-evZas igfRis sc-ewDcJz jtYAuB"
                  color="GREY_2"
                >
                  Fi is a money management platform that re-imagines the banking experience in India. The Fi account, in partnership with Federal Bank, is a digital bank account that gives you the fastest way to open a bank account online.
                </p>
                <p
                  class="sc-evZas igfRis sc-ehMyHa bxRvfH"
                  color="GREY_2"
                >
                  You can do everything from the Fi App, including p2p payments, fund transfers, bill payments, and more, with features to automate every action. You also get a Fi-Federal co-branded Debit Card, spends insights and tools to grow your investment and earn rewards.
                </p>
              </div>
              <div
                class="sc-eCYdqJ iPRnVa sc-djvmMF bnhvWu"
                color="GREY_2"
              >
                Disclaimer: You may have noticed some brand logos used on this website to indicate where you, as a user, may or may not have spent money. We don’t endorse these brands. Nor do these brands endorse us. The logos of the specific brands are owned by them.
              </div>
              <div
                class="sc-cUEIKg egtcEd"
              >
                <div
                  class="sc-eCYdqJ gibgTe sc-kItwNn bBmJvR"
                  color="TEXT_GREY_1"
                >
                  © epiFi Wealth Pvt. Ltd. 2022
                </div>
                <div
                  class="sc-eCYdqJ gibgTe sc-kItwNn bBmJvR"
                  color="TEXT_GREY_1"
                >
                  © epiFi Technologies Pvt. Ltd. 2022
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-eKBdFk iPGhao"
    />
    <div
      class="sc-jOhDuK sc-hlnMnd vEAce fyIvxi"
    >
      <div
        class="sc-bUbCnL iygcBD"
      >
        <a
          href="https://play.google.com/store/apps/details?id=com.epifi.paisa&referrer=utm_source%3Dfimoney"
          id="footer_playstore_link"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="sc-kIKDeO fjNkXd"
          >
            <img
              alt="logo"
              class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
              loading="lazy"
              src="play-store_black.png"
            />
          </div>
        </a>
        <a
          href="https://apps.apple.com/in/app/fi-money-digital-banking/id1531564767"
          id="footer_appstore_link"
          rel="noreferrer"
          target="_blank"
        >
          <div
            class="sc-hNKHps cvMsDD"
          >
            <img
              alt="logo"
              class="sc-gsnTZi sc-dkzDqf krIvpr hKVGsE"
              loading="lazy"
              src="app-store_black.png"
            />
          </div>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
